scripts/monotone-framework-lattice.py

```diff
@@ -0,0 +1,230 @@
+import argparse
+import sys
+from collections import defaultdict
+from pygraphviz import AGraph
+
+import importlib
+monotone_framework = importlib.import_module("monotone-framework")
```

Comment:
Uh? Why?

Reply:
you cannot import modules with "-" in the name using normal import statements

maybe it would be better to rename the old script, but I didn't want to mess with the existing stuff

Reply:
Go ahead and rename